README.md

```diff
@@ -115,8 +115,17 @@ You can specify a config object, properties and default values are:
     // generate a hostname so we can track this connection on the broker (rabbitmq management plugin)
     hostname: process.env.HOSTNAME || process.env.USER || uuid.v4(),
 
-    // the transport to use to debug. if provided, arnavmq will show some logs
-    transport: utils.emptyLogger
+    // Deprecated. Use 'logger' instead. the transport to use to debug. if provided, arnavmq will show some logs
```

Comment:
maybe we also could print a deprecation message to the console one time (if it's still being used). `util.deprecate` may help with that I think...

Reply:
The `util.deprecate` wraps a function, but this is just a config field, so I'll call `process.emitWarning` with a `DeprecationWarning` manually in case the transport is set.